quickstart/database/seeds/TasksTableSeeder.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class TasksTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+    	$faker = Faker\Factory::create();
+
+    	$limit = 10;
+    	for ($i=0; $i < $limit; $i++) { 
```

Comment:
convention: sau giữa các toán tử là 1 space: $i = 0, $i ++. 
Có thể dùng $i < 10, không nhất thiết phải tạo biến